src/scripts/eastern_kingdoms/dun_morogh/gnomeregan/gnomeregan.cpp

```diff
@@ -35,33 +35,33 @@ EndContentData */
 
 enum
 {
-    SAY_START                   = -1090000,
-    SAY_INTRO_1                 = -1090001,
-    SAY_INTRO_2                 = -1090002,
-    SAY_INTRO_3                 = -1090003,
-    SAY_INTRO_4                 = -1090004,
-    SAY_LOOK_1                  = -1090005,
-    SAY_HEAR_1                  = -1090006,
-    SAY_AGGRO_1                 = -1090007,
-    SAY_CHARGE_1                = -1090008,
-    SAY_CHARGE_2                = -1090009,
-    SAY_BLOW_1_10               = -1090010,
-    SAY_BLOW_1_5                = -1090011,
-    SAY_BLOW_1                  = -1090012,
```

Comment:
Script text -1090012 has emote = 25 (Point), while broadcast text id 4137 which u replace it with currently has no emote, but you do not update the broadcast text in your migration.